Amplify/Categories/DataStore/Query/QuerySortInput.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// A simple enum that holding a sorting direction for a field that can be applied to queries.
+public enum QuerySortBy {
+    case ascending(CodingKey)
+    case descending(CodingKey)
+}
+
+public struct QuerySortInput {
+    public let inputs: [QuerySortBy]
+
+    public init(_ inputs: [QuerySortBy]) {
+        self.inputs = inputs
+    }
+
+    /// Creates a `QuerySortInput` in an expressive way, enabling a short
+    /// and developer friendly access to an instance of `QuerySortInput`.
```

Comment:
Suggest you add a usage example to this doc to show how friendly it is to access :)

Reply:
Hahaha, Added something comments